libs/utils/src/lib/models/logicAppsV2.ts

```diff
@@ -100,6 +100,14 @@ export interface OpenApiOperationInputs extends RetryableActionInputs {
   parameters?: any;
 }
 
+export interface HybridTriggerOperationAction extends Trigger {
+  inputs: HybridTriggerConnectionInfo;
```

Comment:
Why do you call it action and trigger when this is just trigger.. and this does not seem correct you have other properties also inside inputs no? like the schema

Reply:
Added the other missing properties